easybuild/easyconfigs/j/json-fortran/json-fortran-8.3.0-GCCcore-11.3.0.eb

```
@@ -0,0 +1,33 @@
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'json-fortran'
+version = '8.3.0'
+
+homepage = 'https://github.com/jacobwilliams/json-fortran'
+description = "JSON-Fortran: A Modern Fortran JSON API"
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/jacobwilliams/json-fortran/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['5fe9ad709a726416cec986886503e0526419742e288c4e43f63c1c22026d1e8a']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.24.3'),
+]
+
+configopts = [
+    '-DUSE_GNU_INSTALL_CONVENTION=TRUE',
+]
```

no need for this to be a list?
```suggestion
configopts = '-DUSE_GNU_INSTALL_CONVENTION=TRUE'
```